src/main/java/microsoft/exchange/webservices/data/EwsXmlReader.java

```diff
@@ -1121,4 +1121,17 @@ private static boolean isNullOrEmpty(String namespacePrefix) {
 
   }
 
+  /**
+   * Gets the error message which happened during {@link #readValue()}.
+   *
+   * @param details details message
+   * @return error message with details
+   */
+  private String getReadValueErrMsg(final String details) {
+    return String.format(
```

Comment:
could't this also been used with simple string concatenation since this would also be faster?